### PR TITLE
Create new-site.md

### PR DESCRIPTION
**Pull request by theo-cox**
[m]: create blog for SCQA about site transition

### CODE REVIEW

**Comment by theo-cox** [comment]
@rufuspollock SCQA blog here

File: content/notes/new-site.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+type: Blog
+---
+
+
+You might have noticed that the open revolution website has evolved. We realized that we had a lot of material (and even more thinking) which we hadn’t published anywhere, and that there was a clear need for more generalized and practically focused content. We created the new diginomics site to address this need. The site provides grounding in the economics of digital (information) economy and toolkits of concepts and templates for solving key micro and macro challenges e.g. sustaining data sharing in X area (e.g. carbon data) or addressing tech monopolies.
+
+This blog lays out our thinking behind the evolution in the SCQA format.
+
+
+# Situation
```
Comment (rufuspollock) [inline content/notes/new-site.md:14]: `## Context: Open Revolution is five years old and much has changed`

File: content/notes/new-site.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+The openrevolution.net site and book have existed for some time. The content has been well received but the book tells a high level story. The existing openrevolution.net site focuses on the book; it’s currently just “the book website”. 
+
+The topics covered by the book continue to become ever more visible and mainstream (Google, Apple, Facebook, Amazon etc are now clearly semi “bad guys”). We are getting approached by people involved in various projects related to data sharing who want insight into the economics and incentives to underpin this; understanding is limited and there is a need.
+
+We have other material both published (e.g. imedproject) or unpublished that is relevant, which could be helpful if it was more accessible to people.
+
+
+# Complication
```
Comment (rufuspollock) [inline content/notes/new-site.md:22]: `## The Need and Opportunity: there's a broader topic to address and we already have the material`

File: content/notes/new-site.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Current material is also so-so could be easily improved:
+
+
+* We have detailed answers (and even material) for the above and it isn’t on the site (e.g. book not footnoted, detailed models for particular areas).
+* Material on the site isn’t immediately accessible or useful in many applied contexts.
+* The book offers a general solution rather than specific examples of how the ideas could work in different scenarios.
+
+
+# Question
```
Comment (rufuspollock) [inline content/notes/new-site.md:33]: `## How can we evolve this project?

File: content/notes/new-site.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+What do we build which incorporates our existing work and thinking in a way which makes it accessible and useful for diverse people and challenges, and which does not require the use of significant extra resources beyond those we have spare in-house?
```
Comment (rufuspollock) [inline content/notes/new-site.md:34]: How can we evolve this project?  ...

**Review by rufuspollock — CHANGES_REQUESTED** [review]
This was already good. Changes are relatively minor - they are about "making it read" better and some refinement of intent and tone.

I've left detailed comments for your edification (haven't quite finished - and think easy to edit directly now).

For now i'm going to copy your text, apply changes and commit - and you can see the relative changes if you wish.

**Pull request closed without merging**